frontend/src/Components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,76 @@
+import { Icon } from '@iconify/react';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { getGang } from '~/api';
+import { GangDto } from '~/dto';
+import { KEY } from '~/i18n/constants';
+import styles from './Breadcrumb.module.scss';
+import { BreadcrumbTitles } from './BreadcrumbTitles';
+
+export function Breadcrumb() {
+  const { t } = useTranslation();
+  const location = useLocation();
+  const segments = location.pathname.split('/').filter(Boolean);
+  const baseUrl = 'http://localhost:3000';
+  const [gang, setGang] = useState<Partial<GangDto>>({});
+  const gangId = useParams().gangId;
+
+  useEffect(() => {
+    if (gangId) {
+      getGang(gangId)
+        .then((gang) => {
+          setGang(gang);
+        })
+        .catch((error) => console.error('Error fetching gang:', error));
+    }
```

Comment:
Anbefaler guard pattern (early return)
```suggestion
    if (!gangId) return
      getGang(gangId)
        .then((gang) => {
          setGang(gang);
        })
        .catch((error) => console.error('Error fetching gang:', error));
```

Reply:
Utdatert, resolve?